src/lib/util/flag-resolver.ts

```diff
@@ -22,13 +23,16 @@ export default class FlagResolver implements IFlagResolver {
         const flags: IFlags = { ...this.experiments };
 
         Object.keys(flags).forEach((flagName: IFlagKey) => {
-            if (!this.experiments[flagName]) {
-                if (typeof flags[flagName] === 'boolean') {
+            const flag = flags[flagName];
+            if (typeof flag === 'boolean') {
+                if (!flag) {
```

Comment:
Wouldn't this prevent the flag to be evaluated again? If it's turned true once, it seems you can't disable it because you'll skip the `isEnabled` evaluation

Reply:
If I understood the original reasoning, our `flag` here is the default value we get from either `experimental` or `server-dev`, not the current state of the flag. 

What we're saying here is, unless one those two places above override the flag by setting it as something `truthy` (e.g. env variable), we want to fetch it from the external resolver.

Reply:
Ok, I saw it now, and flags is a local variable of this method